browser-test/src/support/admin_programs.ts

```diff
@@ -312,11 +312,26 @@ export class AdminPrograms {
     }
   }
 
+  async openQuestionBank() {
+    await this.page.click('button:has-text("Add a question")')
+    // wait for animation to finish
+    await this.page.waitForTimeout(1000)
```

Comment:
suggestion (blocking): if the animation is 1s please extract 1000 here to a constant

Reply:
Animation is 150ms: https://tailwindcss.com/docs/transition-property

I put 1s as upper limit to hopefully avoid flakiness. I can expand comment here. But don't see how having it as constant would be useful yet.

Reply:
Ah got it, a comment would be good then

Reply:
Done.